themes/aframe/layout/blog.ejs

```diff
@@ -8,29 +8,59 @@
 <div class="content content--blog copy post">
   <div class="copy__wrap">
     <% if (page.blog_index) { %>
-
       <h1 class="page__title"><%- config.blog_title %></h1>
 
       <%- page.content %>
 
       <% site.posts.sort('date', -1).each(function (post) { %>
         <article class="post">
-          <h2 class="post__heading"><%- post.title %></h2>
-          <h4 class="post__timestamp"><%- post.date.format('MMM D[,] YYYY') %></h4>
-          <div class="post__excerpt"><%- post.excerpt %></div>
+          <a href="<%- page_url(post.path) %>">
+            <h2 class="post__heading"><%- post.title %></h2>
+          </a>
+          <h4 class="post__timestamp"><%- blog_date(post.date) %></h4>
+
+          <% if (post.twia) { %>
+            <p>What's up with A-Frame for the week of <%- blog_date_subtract_week(post.date) %> to <%- blog_date(post.date) %>.</p>
+
+            <div class="post__projects">
+              <% post.twia.projects.forEach(function (project, i) { %>
+                <% // Display even number of projects. %>
```

Comment:
eh, you don't think we should instead just use CSS styles to hide (based on the layout)? maybe on desktop allow thirds?

perhaps, we should actually just max out at 2, not sure. it worked okay most of the time with the flexbox grid styles I used for the Tumblr.

it's probably fine for now, but could you clarify the comment a bit: `// Showing only thumbnails in groups of two for post previews so they don't look awkward.`